app/views/layouts/application.slim

```diff
@@ -49,8 +49,13 @@ html
                     span.text-warning Read-Only Mode Active
               li.nav-item class=('active' if on_page[:faq])
                 a.nav-link.px-3 href=faq_path FAQ
-              = form_for(:game, method: :get, url: games_path, html: {class: 'form-inline ml-2 flex-grow-1', role: 'search'}) do |f|
+              = form_for(:search, method: :get, url: search_path, html: {class: 'form-inline ml-2 flex-grow-1', role: 'search'}) do |f|
                 .input-group
+                  select.input-group-addon name='search_type'
+                    option value='game' selected=('true' if @search_type == 'game')
+                      | Game
+                    option value='user' selected=('true' if @search_type == 'user')
+                      | User
```

Comment:
Is it possible to ditch the select boxes and always search both users and games, displaying both types of results together on the search results page? imo the fewer clicks/keypresses/decisions users have to make, the better.

Reply:
Are you ok with this being in different tabs like the categories on the games page?  I feel like having everything in one long list would get a bit cluttered especially if you plan on adding search to anything else in the future.

Reply:
hmmmm. imo putting them in different tabs is better than the selectors (one less decision to make before performing the search) but still not as good as all-in-one-page (0-1 more clicks to get to the result you want). Can the clutter issue be solved with pagination?

The eventual future I want is like how the new twitch.tv search works, with a few results from each type immediately visible, and you can click "more" on any result type to see a fuller list (see below).

This is obviously way out of scope, but small steps toward it will help.

<img width="411" alt="screenshot 2018-05-02 09 41 58" src="https://user-images.githubusercontent.com/438911/39537025-2c22866a-4ded-11e8-8124-e37be4385ed7.png">

<img width="414" alt="screenshot 2018-05-02 09 54 18" src="https://user-images.githubusercontent.com/438911/39537545-d062a4c0-4dee-11e8-8e27-3f6d5249093a.png">